README.md

```diff
@@ -0,0 +1,32 @@
+# valid-github-username
+
+## Installation
+Make sure you have node installed. Then clone the repo.
+
+    git clone https://github.com/andrewda/valid-github-username.git
+    
+## Usage
+Use `npm test` to use the program.
```

Comment:
Nice job on the readme! It seems mostly correct, however: `npm test` runs the test script for the program, and not the actual code that checks if the username is valid. 

The actual code can be found in `index.js`. It's just a single function that takes a `name` parameter. My suggestion would be to add a bit about importing and running said function too. 

Also, I think it would be best if you squashed your 2 commits into 1. Overall, well done!